tests/MySQLDatatypeTest.php

```diff
@@ -51,6 +51,28 @@ public function testInvalidNumericLengths($length)
     {
         try {
             new MySQL("NUMERIC", ["length" => $length]);
+            new MySQL('NUMERIC', [
+                'length' => [
+                    'numeric_precision' => '38',
+                    'numeric_scale' => '0'
+                ]
+            ]);
+            new MySQL('NUMERIC', [
+                'length' => [
+                    'numeric_precision' => '38',
+                    'numeric_scale' => '38'
+                ]
+            ]);
+            new MySQL('NUMERIC', [
```

Comment:
Tomuto neozumiem ... staci ze jedno z volani vyhodi exception a spadne to .... takze ostatne sa neotestuju .... napr, spadne volanie na riadku `60`, vyhodi sa exception -> volanie na riadku `66` sa ani nespusti .... IMHO tam treba spravit nejaky `data provider` na to pole co je parameter, ... alebo aspon cyklus.

Reply:
nj, já jsem to dal do invalid testu....opravené díky :)